frontend/src/views/producer/dashboard/edition-form/Editor.jsx

```diff
@@ -410,12 +429,13 @@ class Editor extends React.Component {
             </InputField>
           </Row>
           <Row>
-            <InputField label="Age group limit" lightMode horizontal>
+            <InputField label="Ikäsuositus" lightMode horizontal>
               <TagPillGroup
                 //multiple //@TODO: toogle this once EventModal gets updated
                 highlightColor={this.internalData.eventDraft.themeColor}
                 onChange={this.onAgeGroupChange}
-                value={this.internalData.eventDraft.ageGroupLimit}
+                value={this.internalData.eventDraft.ageGroupLimits}
+                multiple
                 tags={[
```

Comment:
Here's one more array to put into a constant...

Reply:
The `tags` value..

Reply:
https://trello.com/c/PFoJDOag/102-refactor-move-all-static-datas-to-a-constant-file

Reply:
Lets have a refactor session for this.